src/App.css

```diff
@@ -1,51 +1,51 @@
-.App {
-  text-align: center;
+:root {
+    --primary: #389E6F;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.bg-primary {
+    background-color: var(--primary);
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
```

Comment:
Prefer proper naming conventions for CSS classes.
Always use uppercase for naming classes